src/Microsoft.Health.Dicom.SqlServer/Registration/DicomSqlServerRegistrationExtensions.cs

```diff
@@ -64,6 +64,8 @@ public static class DicomSqlServerRegistrationExtensions
             .AddSqlQueryStores()
             .AddSqlWorkitemStores();
 
+        services.AddSingleton<StoreMeter>();
```

Comment:
This is kind of confusing since `Store` is an overloaded term in our codebase. The core logic for the `Store` transaction is what you find in `Microsoft.Health.Dicom.Core.Features.Store`, and that's where the meter should be added. This module is a data store.

Reply:
Thanks Josiah. I have made the required changes